extensions/amp-form/0.1/amp-form.js

```diff
@@ -104,10 +121,21 @@ export class AmpForm {
    * @private
    */
   setState_(state) {
+    this.state_ = state;
     for (const key in FormState_) {
       this.form_.classList.remove(`amp-form-${FormState_[key]}`);
     }
     this.form_.classList.add(`amp-form-${state}`);
+
+    if (state == FormState_.SUBMITTING) {
```

Comment:
You mean the `forEach`? Yeah I don't mind that. Is this just to get rid of the double `forEach` statements?